DREAM/Optimization/tsgOptimizationUtils.hpp

```diff
@@ -57,6 +57,16 @@
 
 namespace TasOptimization {
 
+/*! \internal
+ * \ingroup OptimizationUtil
+ *
+ * Stores information about the run of an optimization algorithm.
+ */
+struct OptimizationStatus {
+    int num_iterations;
```

Comment:
Is this the max or the actual iterations?

Use better names:
* performed_iterations or done_iterations
* max_iterations

Reply:
This is the actual iterations, so I have renamed it to `performed_iterations`.